src/jit/core_templates.expr

```diff
@@ -171,12 +193,25 @@
       (carg (tc) ptr)
       (carg $1   int)) ptr_sz))
 
+(template: coerce_ns
+  (call (^func &MVM_coerce_n_s)
+    (arglist
+      (carg (tc) ptr)
+      (carg $1 num)) ptr_sz))
+
 (template: coerce_si
   (call (^func &MVM_coerce_s_i)
     (arglist
       (carg (tc) ptr)
       (carg $1   ptr)) int_sz))
 
+(template: coerce_sn
+  (calln (^func &MVM_coerce_s_n)
+    (arglist
+      (carg (tc) ptr)
+      (carg $1   ptr))))
```

Comment:
This shouldn't be `(carg $1   ptr)) num_sz))`?

Reply:
> This shouldn't be `(carg $1 ptr)) num_sz))`?

Note this uses the new `calln`, not `call`